client/src/components/posting-page/JobPosting.js

```diff
@@ -3,7 +3,7 @@ import { connect } from 'react-redux'
 import * as actions from '../../redux/actions/actions'
 import PropTypes from 'prop-types'
 import { Link, withRouter } from 'react-router-dom'
-import { Button, Chip } from '@material-ui/core/'
+import { Button, Chip, Snackbar } from '@material-ui/core/'
```

Comment:
Andrea found [information about how to minimize bundle size](https://material-ui.com/guides/minimizing-bundle-size/)

Reply:
Ok, I think it's still like this in other files, but we'll change it as we go I guess?